### ISSUE TITLE
.opml Format

### ISSUE THREAD

**Issue by Mentsh** [dsopas/assessment-mindset#10]
Love the mind map! What a wonderful resource. I was trying to make something similar but did not know where to start nor what platform to organize my thoughts and TTPs.  I have a request- I use simplemind (mind mapping) and I would love to import this into it - however I would need a paid subscription to xmind to export to .opml which can then be imported to simpleminded.

Would you be willing to provide that export?

Thank you!

**Comment by dsopas** [comment]
Yes of course next version will have .opml export file!
Thanks for using it!

**Comment by Mentsh** [comment]
It would appear Links will not be included in .opml -- I tested with one of your older pushes.  Looks like I will have to manually add them unless you know another way.

Thanks.

**Comment by Mentsh** [comment]
Let me know what you think before I do a pull request.. mine has some drastic changes.... and you may not want to modify it like I have...  I will not take it personally as everyone will like something different. Feel free to just pick and choose what you want out of mine.. Mostly added some tools. Added the OWASP Mobile app top 10. Added configuration flaws topic and added a few injection flaws. 
 
also I was not able to find anything definitive on the web so I took a poll on my local security meet up slack and everyone said Subdomain takeovers are network attacks... so I moved that accordingly... 


Again change from .
[assessment_mindset.txt](https://github.com/dsopas/assessment-mindset/files/4195290/assessment_mindset.txt)
txt to .opml

**Comment by dsopas** [comment]
@Mentsh I think you did an awesome job. Feel free to PR. 
Also I'm adding your Name / Handle (link to twitter) on the Thanks section. Can you provide me the details for that?

**Comment by Mentsh** [comment]
Sure, 
I go by Mentsh
https://twitter.com/M3n7sh


I moved somethings around yesterday.. Such as Email recon > logical > Employees  
I also moved a few other things under Employees | License plate, Social Media of employees added vs Company Social Media,  people search.... Added more sub-Categories to Office Physical and Logical... and around 40+ new subtopics.  Still a work in progress
Let me know how this one sets with you.  I can do a Pull request with the other one or with this version 3... Your choice 
![assessment_mindset_v3](https://user-images.githubusercontent.com/33880194/74476955-05de9b80-4e70-11ea-9740-60aed71f285d.png)


To be Transparent - So that I can modify and share (Because I may go in a direction you don't like) I am going to create a Repo on my GitHub with my latest - but I will link to yours. We can do pull requests as you see fit. I hope you are ok with that... I have really modified the great one you started.
